data/espers.py

```diff
@@ -141,23 +141,23 @@ def get_spell():
                 learn_rate = Esper.LEARN_RATES[learn_rate_index]
                 esper.add_spell(get_spell(), learn_rate)
 
-    def remove_all_ultima(self):
-        ultima_id = self.spells.get_id("Ultima")
-        for esper in self.espers:
-            if esper.has_spell(ultima_id):
-                esper.remove_spell(ultima_id)
+    def remove_all(self):
```

Comment:
⛔  Bit of a misnomer, perhaps something like `remove_flagged_learnables` would work

Reply:
good call; renamed